features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -659,6 +659,8 @@
   "authentication.adaptive.http_connections.connection_timeout": "$ref{authentication.adaptive.http_connections.default_timeout}",
   "authentication.adaptive.http_connections.read_timeout": "$ref{authentication.adaptive.http_connections.default_timeout}",
   "authentication.adaptive.http_connections.request_timeout": "$ref{authentication.adaptive.http_connections.default_timeout}",
+  "authentication.adaptive.call_choreo.retry_count.token_request_retry_count": "2",
```

Comment:
Do we need to have a separate `call_choreo` path? cannot we reuse the `http_connections` path

Reply:
That config is related to connection time out and the one we are introducing is a request retry.
That's why had a separate section

Reply:
It is not good to have too many first-class paths for the same component/functionality
Some thing like `authentication.adaptive.http_connections.token_request_retry_count` would do

Reply:
Ack